sdk/lib/convert/json.dart

```diff
@@ -477,6 +555,23 @@ class _JsonUtf8EncoderSink extends ChunkedConversionSink<Object?> {
 ///
 /// A JSON input must be the JSON encoding of a single JSON value,
 /// which can be a list or map containing other values.
+/// 
+/// Example:
+/// ```dart
+/// const JsonDecoder decoder = JsonDecoder();
+///
+/// const String strRawJson =
```

Comment:
Nit: don't think this needs "raw" in the variable name

Reply:
fixed